docs/pages/workflow/customizing.md

```diff
@@ -37,12 +37,12 @@ If you want to make static changes to your native project files like the iOS `In
 
 If you've decided that you want to roll your app back to being fully managed (no iOS and Android projects in your project directory), you can checkout your most recent commit before executing `expo run:[ios|android]`, then run `npm install` again to restore the state of your `node_modules` directory.
 
-## Releasing apps with custom native code to production
+## Developing apps with custom native code
 
-The classic `expo build` command does not support custom native code. When you're ready to ship your app, you can [build it with EAS Build](/build/introduction.md) or archive and sign it locally.
+Once you have customized the native code in their project, you can use the [`expo-dev-client`](/clients/introduction.md) package to create a custom development client and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a custom client for your managed or bare workflow by following [our guide](/clients/getting-started.md).
```

Comment:
```suggestion
Once you have customized the native code in your project (by installing a new package outside the SDK, or by writing your own), you can use the [`expo-dev-client`](/clients/introduction.md) package to create a custom development client and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a custom client for your managed or bare workflow by following [our guide](/clients/getting-started.md).
```

Reply:
🙏  I think the rest of the page should be providing the context around how to customize the code so we don't need the callout.
Fixing the "their" though!